src/site/_filters/i18n.js

```diff
@@ -76,4 +76,13 @@ const i18n = (path, locale = 'en') => {
   }
 };
 
-module.exports = {i18n};
+/**
+ * Infer the page locale using page's filePathStem.
```

Comment:
Would you mind adding a note that this is only needed when we need to use the i18n filter inside of the implementation of a shortcode? Otherwise the 11tydata.js file at the root of each language directory should set the locale so normally you can do `'i18n.some.thing' | i18n` and it'll "just work"

Reply:
I actually see this as a helper function that can be used in different types of context. If you already have a locale, either from the top of the data cascade, or passed as a param, then great and don't use this function. But I'd rather not define it's usage here. wdyt?